tests/std/tests/P0355R7_calendars_and_time_zones_formatting/test.cpp

```diff
@@ -218,6 +218,21 @@ constexpr void print(Str str) {
     }
 }
 
+template <typename CharT>
+void test_duration_formatter() {
+    empty_braces_helper(seconds{5}, STR("5s"));
+    empty_braces_helper(minutes{7}, STR("7min"));
+    empty_braces_helper(hours{9}, STR("9h"));
+    empty_braces_helper(days{2}, STR("2d"));
+    empty_braces_helper(-seconds{5}, STR("-5s"));

```

Comment:
I think we should have additional test cases for floating-point and custom unit suffix here.